tests/api/tests.py

```diff
@@ -33,7 +33,7 @@
 DEFAULT_SERVER = 'localhost'
 
 PORT_ENV_VAR_NAME = 'PDC_CLIENT_TEST_SERVER_PORT'
-DEFAULT_PORT = 8378
+DEFAULT_PORT = 8278
```

Comment:
Something wrong with port 8378?

Reply:
Yes, it always should me below error with 8378.
ERROR: setUpClass (tests.api.tests.PDCClientTestCase)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/chcao/cc-build/pdc-client-source/pdc-client/tests/api/tests.py", line 248, in setUpClass
    cls.server = HTTPServer((cls.server_name, cls.port), _MockPDCServerRequestHandler)
  File "/usr/lib64/python2.7/SocketServer.py", line 417, in __init__
    self.server_bind()
  File "/usr/lib64/python2.7/BaseHTTPServer.py", line 108, in server_bind
    SocketServer.TCPServer.server_bind(self)
  File "/usr/lib64/python2.7/SocketServer.py", line 431, in server_bind
    self.socket.bind(self.server_address)
  File "/usr/lib64/python2.7/socket.py", line 228, in meth
    return getattr(self._sock,name)(*args)
error: [Errno 98] Address already in use

Reply:
The test process was probably not closed correctly. The thread with fake server can use some improvements.

Reply:
Yes, I think so.